Tools/SwiftBrowser/Source/BrowserViewModel.swift

```diff
@@ -106,4 +208,18 @@ final class BrowserViewModel {
             Self.logger.error("Failed to export PDF: \(error)")
         }
     }
+
+    func didImportFiles(result: Result<[URL], any Error>) {
+        precondition(currentFilePicker != nil)
+
+        switch result {
+        case let .success(urls):
+            currentFilePicker!.completion(.ok(urls))
+
+        case .failure(_):
```

Comment:
You can also drop the `()` entirely then:
```suggestion
        case .failure:
```